README.md

```diff
@@ -18,7 +18,13 @@ $ npm i skyverge/sake
 $ npm update skyverge/sake
 ```
 
-**Note:** Sake is currently not versioned, but a specific commit may be provided when installing, such as `skyverge/sake#0d5ec6622a6845f695d9e8ed04f4db08bd0e4986`.
+> [!NOTE]
+> Sake is currently not versioned, but a specific commit may be provided when installing, such as `skyverge/sake#0d5ec6622a6845f695d9e8ed04f4db08bd0e4986`.
+
+> [!IMPORTANT]
+> Since commit [18cc1f611af69e9d38d4b0904a41556f6719fbfe](https://github.com/godaddy-wordpress/sake/commit/18cc1f611af69e9d38d4b0904a41556f6719fbfe), the WooCommerce deployment process updated.
```

Comment:
Just for readability, I think we can get away with only displaying the first 7 characters of the commit hash.
```suggestion
> Since commit [18cc1f6](https://github.com/godaddy-wordpress/sake/commit/18cc1f611af69e9d38d4b0904a41556f6719fbfe), the WooCommerce deployment process updated.
```

Reply:
Good call